.github/labeler.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: Labeler
+
+libraries:
+  - lib/**/*
+  - include/grass/*
+  - include/grass/**/*
+  - python/**/*
+
+module:
+  - db/**/*
+  - display/**/*
+  - general/**/*
+  - imagery/**/*
+  - misc/**/*
+  - ps/**/*
+  - raster/**/*
+  - raster3d/**/*
+  - scripts/**/*
+  - temporal/**/*
+  - vector/**/*
+
+# helperscripts:
+#  - utils/*
+#  - utils/**/*
+
+GUI:
+  - gui/**/*
+
+Windows:
+  - mswindows/**/*
+
+macOS:
+  - macosx/**/*
+
+Linux:
+  - rpm/**/*
+  - singularity/**/*
+
+docker:
+  - docker/**/*
+
+# docs:
+#   - doc/**/*
+# manual:
+#   - man/**/*
```

Comment:
We have manual and HTML.

Reply:
Perhaps we can use the (existing) label `manual` for documentation in a broad sense like:
 
```
manual:
  - man/**/*
  - doc/**/*
```

Reply:
Or rename manual to documentation or docs. That would make sense to me.

Reply:
`docs` is short and concise! See also https://github.com/OSGeo/grass/pull/531#discussion_r422619437 and below...

Reply:
updated to use `docs`

Reply:
Sounds good. I agree with my 2020 self.